UnitySetup/UnitySetup.psm1

```diff
@@ -754,6 +754,8 @@ function Get-UnityProjectInstance {
    Force operation as though $PWD is not a unity project.
 .PARAMETER ExecuteMethod
    The script method for the Unity Editor to execute.
+.PARAMETER AdditionalArguments
+   Additiong arguments for Unity or your custom method
```

Comment:
It would be awesome to add an example to the docs as well showing how to use this.